### ISSUE TITLE
vignette needs a lot of work

### ISSUE THREAD

**Issue by beanumber** [baumer-lab/wikitablr#10]
This needs to be heavily revised based on the new codebase.

**Comment by rporta23** [comment]
Should we wait to revise it until we have all of the functions working more smoothly? It seems like it might need to change based on whether we include methods for reading tables and lists for the functions.

**Comment by beanumber** [comment]
Yes, this can wait, but it can also be useful to write down how you think the package should be used, and then go about writing the code to make that possible.
